lib/charms/layer/mongodb.py

```diff
@@ -61,11 +60,11 @@ def install(self):
 
     def uninstall(self):
         apt_purge(self.packages())
-        _run_apt_command(['apt-get', 'autoremove', '--purge', '--assume-yes'])
+        subprocess.check_call(['apt-get', 'autoremove', '--purge', '--assume-yes'])
 
     def configure(self, config):
         cfg = {self.config_map.get(k, k): v
-               for k, v in iter(config.items()) if k in self.config_options}
+               for k, v in iter(config.items()) if v and k in self.config_options}
```

Comment:
Because "False" is not a valid value to put in the mongodb conf. Neither is "None".

tbh I think the configuration file writing in this charm is pretty broken; this PR just gets the layer to the point where mongodb actually starts up, which is progress.